### ISSUE TITLE
Map skipping to Record Collection after Accessing Nested Arrays 

### ISSUE THREAD

**Issue by the-thief** [freeCodeCamp/freeCodeCamp#12436]
**Map skipping to Record Collection** 

#### Challenge Name

[Record Collection](https://www.freecodecamp.com/challenges/record-collection)

#### Issue Description

The FCC map skipped to Record Collection in the Advanced Algorithm Scripting section after I completed [Accesssing Nested Arrays](https://www.freecodecamp.com/challenges/accessing-nested-arrays) in Basic Javascript. I wasn't aware that this was out of order until someone else posted on the forum about this issue.
[Record Collection - it was very hard but I worked on it accidentally](https://forum.freecodecamp.com/t/record-collection-it-was-very-hard-but-i-worked-on-it-accidentally/71913)
I checked my profile after reading this post and saw that Record Collection was way down. I noticed this challenge was above the level I had been working on and also that the next few challenges were a lot easier in comparison.

I added a screenshot of my profile which shows I completed _[Accessing Nested  Arrays](https://www.freecodecamp.com/challenges/accessing-nested-arrays)_ then _Record Collection_ on December 22. Then it jumped back into Basic Javascript with [_Iterate Odd Numbers With a For Loop_](https://www.freecodecamp.com/challenges/iterate-with-javascript-for-loops) on December 23. The original poster in the thread above thinks he might have clicked on Record Collection by mistake. I don't recall doing that and for the most part follow the path prescribed by FCC. Other posters also noticed they skipped to that challenge.

This is my first time posting on Github and only doing so to point out a possible bug. Let me know if I missed something and if this is an actual bug in the system.

Thanks,

#### Browser Information

* Browser Name, Version: Chrome, Version 55.0.2883.87 m
* Operating System: Windows 7 Home Premium - SP1
* Mobile, Desktop, or Tablet: Desktop

#### Screenshot

![image](https://cloud.githubusercontent.com/assets/4421229/21756212/14d4da96-d5ec-11e6-81fc-6eacac7e58d2.png)
**_Profile screenshot showing challenges completed before and after Record Collection._**

![image](https://cloud.githubusercontent.com/assets/4421229/21756195/c8c4d9c6-d5eb-11e6-9879-41a80674a7ac.png)
_**Record Collection Challenge**_

**Comment by raisedadead** [comment]
@the-thief sorry about the delay in getting back to you. Looks like this got lost somehow in the pile of issue that were opened last month. Sincerest apologies.

Now coming to your issue, I can't seem to reproduce this locally or on the live website.

Is there a chance that you happened to visit the challenge from the map or someone else's profile, or a link somewhere in the chat, accidentally? We store the last visited challenge in your browser.

So, unless that was the case, I am uncertain how this happened. Apologies for your inconvenience in any case.

**Comment by the-thief** [comment]
@raisedadead Hi I'm not really sure what went wrong but I know I was working through challenges and went straight to Record Collection. I remember thinking this was way hard but worked on it until I managed to solve it. I don't really look at other camper's maps or profile's so I would eliminate that.

I didn't realize this was even an issue until I saw a thread about it on the forum. That's the only real reason I started this thread, because I saw other campers were having the same problem. Did you see the thread that was started on the forum about this? https://forum.freecodecamp.com/t/record-collection-it-was-very-hard-but-i-worked-on-it-accidentally/71913 I posted this when I opened the issue.

This tells me it's not an isolated issue. Did you test on a new account? One that has not completed this challenge yet?

This is way beyond my level of comfort otherwise I would look at how to solve this.

Thanks,
